templates/publisher/listing.html

```diff
@@ -491,6 +491,7 @@ <h2 class="p-heading--4">Additional information</h2>
 <script>
   window.addEventListener("DOMContentLoaded", function() {
     Raven.context(function () {
+      console.log({{ licenses| tojson }});
```

Comment:
hello console.log :smile:

Reply:
ups, this was not meant to be here...:smile: